pkgs/applications/editors/lapce/_maintainers/plugins_update.nu

```diff
@@ -0,0 +1,134 @@
+cd $env.FILE_PWD
```

Comment:
It is possible to add a nix-shell shebang?

I think it's this (needs testing)
```suggestion
#!/usr/bin/env nix-shell
#! nix-shell -i nushell -p nushell

cd $env.FILE_PWD
```

Reply:
```
#!/usr/bin/env nix-shell
#!nix-shell -i nu -p nushell
```